storage/innobase/btr/btr0scrub.cc

```diff
@@ -268,7 +258,7 @@ int btr_page_needs_scrubbing(
          */
 
         /* if encountering page type not needing scrubbing
-        release reference to table object */
+release reference to table object */
```

Comment:
Oh no, why? Can this be avoided?

Reply:
I'm not sure what happened there... I changed it back and now clang-format leaves it alone.